cmd/kured/main.go

```diff
@@ -645,14 +652,6 @@ func rebootAsRequired(nodeID string, rebooter reboot.Rebooter, checker checkers.
 			}
 		}
 
-		var blockCheckers []blockers.RebootBlocker
```

Comment:
I dont think this was a leak. Have you confirmed with pprof? ( i added PR for enable pprof through config)

since blockCheckers is a new var on every loop the old one should be GCed. 

But the new code is still better so i would keep it as in the PR :)

Reply:
It is NOT a leak. but it annoys my brain. And I don't know if could be a future leak. I will change the commit message.